modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -1209,10 +1209,8 @@ class Session implements ISession {
         if( aborted || cancelled || error )
             return
 
-        CacheDB db = null
-        try {
+        try (CacheDB db = CacheFactory.create(uniqueId, runName).openForRead()) {
             log.trace "Cleaning-up workdir"
```

Comment:
It would make sense to move the `log` out of the `try`, then

Reply:
Great point, done